cves/CVE-2007-0404.yml

```diff
@@ -49,27 +49,39 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A script for compliling messages was made vulnerable because the inputs
+  submitted were not being checked to make sure that they were not malicious in nature.
+  The script job was to make OS level commands and if the inputs were not checked correctly
+  changes could to be made to the system if a targetted input is submitted as an arguememnt or in the .po file
+  with commands. If some of these commands are exectuted hackers can take over the system by giving themselves
+  access to everything possible.
+
+  The fix to this vulenrability was passing the input into a system tool that does this sanitizing
+  and removes any possibly malicious strings or characters in the command like ";" and "$"
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
-bounty:
+bounty: 
   amt:
   announced:
   url:
 reviews: []
-bugs: []
-repo:
+bugs: [407519]
+repo: https://github.com/django/django/commit/518d406e53
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
-fixes:
+fixes: 
 - commit: 518d406e53a7417385a1a2e10bb5110b67d28fac
-  note:
+  note: The os shell line added is used to parse the command line and strips
+   it of backticks, dollar sign usage and any command substitutions and parameter expansions
+   that may run commands that make the system vulnerable
+
 - commit: a132d411c6986418ee6c0edc331080aa792fee6e
-  note:
-vccs: []
+  note: Handles submitting and applying security fix for the changes made in commit 
+    518d406e53a7417385a1a2e10bb5110b67d28fac in a windowns computer set up
```

Comment:
typo: windows

Reply:
I would get rid of "set up", it's not necessary to understand what the sentence is saying